asg/src/accesses/named_access.rs

```diff
@@ -0,0 +1,95 @@
+// Copyright (C) 2019-2021 Aleo Systems Inc.
+// This file is part of the Leo library.
+
+// The Leo library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The Leo library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with the Leo library. If not, see <https://www.gnu.org/licenses/>.
+
+use crate::{ConstValue, Expression, ExpressionNode, FromAst, Identifier, Node, PartialType, Scope, Type};
+use leo_errors::{AsgError, Result, Span};
+
+use std::cell::Cell;
+
+#[derive(Clone)]
+pub struct NamedTypeAccess<'a> {
```

Comment:
Would be good to start documenting code more. I noticed the amount of docs was quite sparse.

Reply:
o oop this file actually should be removed.